main/src/sphexa/sphexa.cpp

```diff
@@ -187,8 +186,8 @@ void printHelp(char* name, int rank)
         printf("%s [OPTIONS]\n", name);
         printf("\nWhere possible options are:\n\n");
 
-        printf("\t--init \t\t Test case selection (evrard, sedov, noh, isobaric-cube, wind-shock) or an HDF5 file "
-               "with initial conditions\n");
+        printf("\t--init \t\t Test case selection (evrard, sedov, noh, isobaric-cube, wind-shock turbulence)\n"
```

Comment:
A comma is missing before turbulence.

Reply:
> Or does a number of steps relative to the last update don't work? Would we need to know the actual number of steps performed until 18 s?

Yes, you need to know the last iteration and add 1, either from the program output or by using one of the scripts in the repository in `scripts/` for HDF5 that will print out the contained steps in the file.